doc/source/whatsnew/v1.1.3.rst

```diff
@@ -47,6 +48,7 @@ Bug fixes
 - Bug in :class:`Series` constructor where integer overflow would occur for sufficiently large scalar inputs when an index was provided (:issue:`36291`)
 - Bug in :meth:`DataFrame.sort_values` raising an ``AttributeError`` when sorting on a key that casts column to categorical dtype (:issue:`36383`)
 - Bug in :meth:`DataFrame.stack` raising a ``ValueError`` when stacking :class:`MultiIndex` columns based on position when the levels had duplicate names (:issue:`36353`)
+- Bug in :meth:`Series.astype` showing too much precision when casting from ``np.float32`` to string dtype (:issue:`36451`)
```

Comment:
this note is duped

Reply:
This was on purpose since at the time I was seeing this as two distinct bugs, but makes more sense to use one note. Will update.